pkgs/dart_services/README.md

```diff
@@ -67,16 +67,12 @@ The server will run from port 8080 and export several JSON APIs, like
 
 To run tests:
 
-`FLUTTER_CHANNEL=stable dart tool/grind.dart test` for unit tests
+`dart tool/grind.dart test`
 
 or:
 
 `grind deploy` for all tests and checks.
 
-dart-services requires the `redis` package, including the `redis-server` binary,
```

Comment:
You might keep some of the redis install info around, perhaps in a different section?